pandas/tests/extension/decimal/test_decimal.py

```diff
@@ -138,6 +138,22 @@ def test_value_counts(self, all_data, dropna):
 
         tm.assert_series_equal(result, expected)
 
+    def test_combine(self):
```

Comment:
can you put this in the base class? (and then if needed skip it for the json tests)

And make use of the fixtures? (so passing `all_data` as an argument to the test function, instead of calling `DecimalArray(make_data())` in the test function

Reply:
That took a bit of work to do, since I need two vectors that are different.  Found discussion at https://github.com/pytest-dev/pytest/issues/2703 about how to do it.

Reply:
that's clever .. :-)